packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -162,7 +162,11 @@ typedef NestedScrollViewHeaderSliversBuilder = List<Widget> Function(BuildContex
 ///
 // See https://github.com/flutter/flutter/issues/54059
 /// Currently, [NestedScrollView] does not support stretching the outer
-/// scrollable, e.g. when using [SliverAppBar.stretch].
+/// scrollable by default, e.g. when using [SliverAppBar.stretch].
+///
+/// If need to support stretching the outer scrollable, you need to set the
```

Comment:
We should not use "you" in our docs, https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#use-the-passive-voice-recommend-do-not-require-never-say-things-are-simple

Reply:
Done